Packs/DSPM/Integrations/DSPM/DSPM.yml

```diff
@@ -0,0 +1,495 @@
+      name: ruleNameIn
+      required: false
+      secret: false
+    - description: 'Exact rule name'
+      isArray: false
+      isOptional: true
+      name: ruleNameEqual
+      required: false
+      secret: false
+    - description: 'List of DSPM tag keys'
+      isArray: true
+      isOptional: true
+      name: dspmTagKeyIn
+      required: false
+      secret: false
+    - description: 'Exact DSPM tag key'
+      isArray: false
+      isOptional: true
+      name: dspmTagKeyEqual
+      required: false
+      secret: false
+    - description: 'List of DSPM tag values'
+      isArray: true
+      isOptional: true
+      name: dspmTagValueIn
+      required: false
+      secret: false
+    - description: 'Exact DSPM tag value'
+      isArray: false
+      isOptional: true
+      name: dspmTagValueEqual
+      required: false
+      secret: false
+    - description: 'List of project IDs'
+      isArray: true
+      isOptional: true
+      name: projectIdIn
+      required: false
+      secret: false
+    - description: 'Exact project ID'
+      isArray: false
+      isOptional: true
+      name: projectIdEqual
+      required: false
+      secret: false
+    - description: 'List of cloud providers ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"]'
+      isArray: false
+      isOptional: true
+      name: cloudProviderIn
+      required: false
+      secret: false
+    - description: 'Exact cloud provider ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"]'
+      isArray: false
+      isOptional: true
+      name: cloudProviderEqual
+      required: false
+      secret: false
+    - description: 'List of affects ["SECURITY", "COMPLIANCE", "GOVERNANCE", "SECURITY_AND_COMPLIANCE", "SECURITY_AND_GOVERNANCE", "COMPLIANCE_AND_GOVERNANCE","SECURITY_AND_COMPLIANCE_AND_GOVERNANCE"]'
+      isArray: true
+      isOptional: true
+      name: affectsIn
+      required: false
+      secret: false
+    - description: 'Exact affect ["SECURITY", "COMPLIANCE", "GOVERNANCE", "SECURITY_AND_COMPLIANCE", "SECURITY_AND_GOVERNANCE", "COMPLIANCE_AND_GOVERNANCE","SECURITY_AND_COMPLIANCE_AND_GOVERNANCE"]'
+      isArray: false
+      isOptional: true
+      name: affectsEqual
+      required: false
+      secret: false
+    - description: 'List of statuses ["OPEN", "CLOSED", "UNIMPORTANT", "WRONG", "HANDLED", "INVESTIGATING"]'
+      isArray: true
+      isOptional: true
+      name: statusIn
+      required: false
+      secret: false
+    - description: 'Exact status ["OPEN", "CLOSED", "UNIMPORTANT", "WRONG", "HANDLED", "INVESTIGATING"]'
+      isArray: false
+      isOptional: true
+      name: statusEqual
+      required: false
+      secret: false
+    - description: 'Sort order'
+      isArray: false
+      isOptional: true
+      name: sort
+      required: false
+      secret: false
+    description: Retrieves risk findings matching the input criteria.
+    name: dspm-get-risk-findings
+    outputs:
+    - contextPath: DSPM.RiskFindings
```

Comment:
its better to add all the fields to the outputs, for example:

```
        - contextPath: DSPM.RiskFindings.id
         description: The finding ID
          type: String
        - contextPath: DSPM.RiskFindings.status
          description: The finding ID.
          type: String
```

**Its really important that the outputs Context Keys should not contain characters like (),[],{},. or spaces.**
you can use the **demisto-sdk generate-outputs** command to create the outputs:
https://docs-cortex.paloaltonetworks.com/r/1/Demisto-SDK-Guide/generate-outputs

Reply:
Sure, we will make necessary changes.